src/app/modules/library/library-project/library-project.component.ts

```diff
@@ -55,4 +65,9 @@ export class LibraryProjectComponent implements OnInit {
       panelClass: 'dialog-md'
     });
   }
+
+  protected selectProject(event: any): void {
+    event.stopPropagation();
+    this.projectSelectionChanged.emit({ checked: event.target.checked, project: this.project });
```

Comment:
Change to ```projectSelectionEvent``` and use "selected" fieldname instead of ```checked``` to match "selection"?
```suggestion
    this.projectSelectionEvent.emit({ selected: event.target.checked, project: this.project });
```